person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  sttr_reader :id
+  sttr_accessor :name, :age
+
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
```

Comment:
- [ ] According to the requirements you should make this method private but I think you forgot to make it as a public method so kindly make it private ✅🔥✨💯